Add deployment instructions (#20)
reddit/reddit-gtm-template: README.md

The official tag template for the Reddit conversion pixel for use with Google Tag Manager

![screenshot](pixel-screenshot.png)

## Deployment

> 1. Submit a template to the Community Template Gallery  |  Google Tag Manager Templates  |  Google Developers  A sample diff can be found here: bumping metadata.yaml for latest release by [woodchuckgym · Pull Request #12](https://github.com/reddit/reddit-gtm-template/pull/12)

Reviewer: ```suggestion
1. Follow [Google's instructions](https://developers.google.com/tag-platform/tag-manager/templates/gallery#update_your_template) for updating the template.  A sample diff can be found in [Pull Request #12](https://github.com/reddit/reddit-gtm-template/pull/12).
```

> 2. Google says the changes will be available in the gallery in 2 to 3 days, but in practice it usually takes only a few hours. Check the "Last Updated" date here: Tag Manager Template Gallery.

Reviewer: ```suggestion
2. Google says the changes will be available in the gallery in 2 to 3 days, but in practice it usually takes only a few hours. Check the "Last Updated" date [here](https://tagmanager.google.com/gallery/#/owners/reddit/templates/reddit-gtm-template).
```
